### PR TITLE
added initial speakers

### PR DESCRIPTION
**Pull request by MikeZTaylor**
@rfeathers Hi Ron, I'm Mike - the front end developer intern at nearForm and I'll be in charge of the website and will be making all changes to it. It's great to meet you.

### CODE REVIEW

File: 2015-digitas/index.html [hunk 1]
```diff
@@ -1223,6 +1223,247 @@ <h3 id="codeofconductLabel">Code of conduct</h3>
...
+        <div class="topic topic-talk">
+
+            <div class="wrapper">
+
+                <div class="row">
+
+                    <div class="col-sm-10">
+
+                <h3 class="title">Richard Rodger</h3>
+                <!--<h3 class="title">Richard Rodger</h3>-->
```
Comment (rfeathers) [inline 2015-digitas/index.html]: Probably shouldn't check in comments - especially along with the uncommented line right above it :)
Not a big deal here, @MikeZTaylor, but a good practice to watch out for.

Comment (MikeZTaylor) [inline 2015-digitas/index.html]: @rfeathers whoops! thanks for that. I'll fix it in the morning and push up the new version.

Comment (ckiss) [inline 2015-digitas/index.html]: @rfeathers yeah you are right about the comments, but this shouldn't be a stopper right? :) @MikeZTaylor will fix this tomorrow and it's fix will be checked in along with other changes. How's that sounds? Cheers

Comment (rfeathers) [inline 2015-digitas/index.html]: Totally not a stopper - I'll merge in a bit

Comment (ckiss) [inline 2015-digitas/index.html]: Excellent, thanks.

File: index.html [hunk 2]
```diff
@@ -2,108 +2,104 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>NodeDay</title>
+
+    <link rel="stylesheet" href="2015-digitas/css/bootstrap.min.css">
+    <link rel="stylesheet" href="2015-digitas/css/app.css" type="text/css" />
+
+    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
+    <!--[if lt IE 9]>
+  <script src="js/html5shiv.min.js"></script>
```
Comment (rfeathers) [inline index.html]: This'll cause a 404

File: index.html [hunk 3]
```diff
@@ -2,108 +2,104 @@
...
+  <script src="js/respond.min.js"></script>
```
Comment (rfeathers) [inline index.html]: 404 here, too

Comment (ckiss) [inline index.html]: Good catch @rfeathers. We'll take care of it in the morning. Mike will let you know when the PR is updated. Cheers.

**Comment by rfeathers** [comment]
Thanks @MikeZTaylor.
In the latest, if looks like the Map URL has reverted and is pointing to Edinburgh again.

**Comment by MikeZTaylor** [comment]
Hi @rfeathers , I fixed that error and it's now pointing to the London location

**Pull request merged**